### PR TITLE
i18n: DE translations

### PR DESCRIPTION
**Pull request by math-GH**
Changes proposed in this pull request:
- translated the labels that were marked as "Todo" in the German (De) folder

How to test the feature manually:
1. switch to German

Pull request checklist:
- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

File: app/i18n/de/conf.php [hunk 1]
```diff
@@ -49,11 +49,11 @@
 			'thin' => 'Klein',
 		),
 		'thumbnail' => array(
-			'label' => 'Thumbnail', // TODO - Translation
-			'none' => 'None', // TODO - Translation
-			'portrait' => 'Portrait', // TODO - Translation
-			'square' => 'Square', // TODO - Translation
-			'landscape' => 'Landscape', // TODO - Translation
+			'label' => 'Vorschaubild',
+			'none' => 'Keins',
```
Comment (Frenzie) [inline app/i18n/de/conf.php]: keins or keine?

File: app/i18n/de/conf.php [hunk 2]
```diff
@@ -200,7 +200,7 @@
 		'navigation_help' => 'Mit der <kbd>⇧ Umschalttaste</kbd> finden die Tastenkombination auf Feeds Anwendung.<br/>Mit der <kbd>Alt ⎇</kbd>-Taste finden die Tastenkombination auf Kategorien Anwendung.',
 		'navigation_no_mod_help' => 'Die folgenden Navigationsverknüpfungen unterstützen keine Modifikatoren.',
 		'next_article' => 'Zum nächsten Artikel springen',
-		'non_standard' => 'Some keys (<kbd>%s</kbd>) may not work as shortcuts.',	// TODO - Translation
+		'non_standard' => 'Einige Tasten (<kbd>%s</kbd>) können nicht als Shortcut verwerndet werden.',
```
Comment (Frenzie) [inline app/i18n/de/conf.php]: typo

File: app/i18n/de/conf.php (hunk 1 above)
Comment (math-GH) [reply app/i18n/de/conf.php]: I would say that "keins" is correkt. 
"kein" would be also correct. 
"keine" is not correct
grafik

File: app/i18n/de/conf.php (hunk 2 above)
Comment (math-GH) [reply app/i18n/de/conf.php]: Thanks!
fixed.

**Review by Frenzie — APPROVED** [review]

**Comment by Alkarex** [comment]
Let me merge https://github.com/FreshRSS/FreshRSS/pull/3841 first

**Comment by Alkarex** [comment]
Done, but now there is a conflict in a single file to resolve

**Comment by math-GH** [comment]
> 
> 
> Done, but now there is a conflict in a single file to resolve

resolved

**Comment by Alkarex** [comment]
There is one remaining TODO :-)

```
Language: de
Untranslated key conf.display.icon.summary - Summary
```

**Comment by Frenzie** [comment]
Merge mistake, was in there as Zusammenfassung originally.

**Pull request merged**